frontend/src/lib/components/MilestoneGroup.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+	import Breadcrumps from '$lib/components/Breadcrumps.svelte';
+	import CardDisplay from '$lib/components/DataDisplay/CardDisplay.svelte';
+	import GalleryDisplay from '$lib/components/DataDisplay/GalleryDisplay.svelte';
+	export let breadcrumpdata: any[] = [];
+	export let milestonedata: any[] = [];
+
+	// FIXME:styling has no business being here... not sure where to put it though given thatparts of it are data dependent
```

Comment:
I think the usual solution to this is to make a component, e.g. we could have our own ProgressBar which takes progress as a prop and is responsible for its styling, etc

Reply:
noted. The progressbar will indeed need it's own thing I think because the flowbite-svelte one is rather limited atm